spec/controllers/calls_controller_spec.rb

```diff
@@ -74,9 +74,11 @@ def setup_active_call_with_new_record(new_record: false)
         allow(@connection).to receive(:legislator).and_return(legislator)
       end
       it "dials the right number" do
-        get :new_connection, 'CallSid': 123
-        xml_response = Oga.parse_xml(response.body)
-        expect(xml_response.css('Dial').text).to eq(@phone)
+        ClimateControl.modify FAKE_CONGRESS_NUMBER: nil do
+          get :new_connection, 'CallSid': 123
+          xml_response = Oga.parse_xml(response.body)
+          expect(xml_response.css('Dial').text).to eq(@phone)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.